index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+    <nav class="mainMenu">
+        <p class ="menuLogo">Ruri Enciso</p>
+        <div class="menuIcon">
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+        </div>
+    </nav>
+    <section class="homeContent">
+        <h1>Hey There. I'm Ruri</h1>
+        <p class="subtitle">I’am a Software Developer</p>
+        <p class="contentPara">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+        <ul>
+            <li>
+                <a><img src="./images/Icons/git.png"></a>
+                <a><img src="./images/Icons/Linkedin.png"></a>
+                <a><img class="peace-icon" src="./images/Icons/peace.png"></a>
+                <a><img src="./images/Icons/twitter.png"></a>
+                <a><img clas="m-icon" src="./images/Icons/m.png"></a>
+            </li>
+        </ul>
```

Comment:
- [ ] Nice job using the `ul` tag to wrap the `li` tag and add your links inside, But we should not add more than one link in a single `li` tag. Please make sure to wrap every link in a single li item. 

### something like 
```HTML 
          <ul>
                <li><a><img src="./images/Icons/git.png"></a></li>
                <li><a><img src="./images/Icons/Linkedin.png"></a></li>
                <li><a><img class="peace-icon" src="./images/Icons/peace.png"></a></li>
                <li><a><img src="./images/Icons/twitter.png"></a></li>
                <li><a><img clas="m-icon" src="./images/Icons/m.png"></a></li>
        </ul>
```